packages/react-components/lib/simple-info.tsx

```diff
@@ -54,7 +56,9 @@ const useStyles = makeStyles((theme) => ({
     display: 'table-cell',
     textAlign: 'end',
     borderBottom: '1px solid',
+    borderBottomColor: theme.palette.divider,
     borderTop: '1px solid',
+    borderTopColor: theme.palette.divider,
```

Comment:
I think the default border color is not the same as the divider color, so doing this would mess up the default material theming.